src/main/java/com/it4045/pokemonviewer/enterprise/dto/Evolution.java

```diff
@@ -7,7 +7,7 @@
 class Evolution {
 
     @SerializedName("num")
-    private String evolutionNumber;
+    private int evolutionNumber;
```

Comment:
The evolution number should have integer value because the variable name itself describes evolution number. Also the name of the evolution has separate variable that is pokemonEvolutionName.

Reply:
Is this the best name for the variable?
"Evolution" is the name of the class, thus this is redundant.
"Number" is a variable type, which does not need to be embedded in the name.

Is there something more specific than evolutionNumber that can be used for this? Something more descriptive.

Perhaps evolutionNumber is the best variable name, but it's worth considering alternatives.